frontend/Panel.js

```diff
@@ -295,6 +295,7 @@ class Panel extends React.Component {
         }}
         extraPanes={extraPanes}
         extraTabs={extraTabs}
+        onViewElementSource={(id, source) => this.viewElementSource(id, source)}
```

Comment:
Should this always be passed? Or only when `this.props.showElementSource` is passed? Note that `viewElementSource` crashes otherwise. For example `showElementSource` doesn't exist in Chrome shell.